gradle/libs.versions.toml

```diff
@@ -13,8 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 [versions]
-adobe-testing-s3mock = "2.17.0"
-amazon-awssdk = "2.26.19"
+adobe-testing-s3mock = "3.9.1"
+amazon-awssdk = "2.28.11"
```

Comment:
Does the AWS sdk need to be updated in this MR? Is this just to align with s3mock?